service/migration_manager.cc

```diff
@@ -903,8 +904,9 @@ future<> migration_manager::announce(std::vector<mutation> mutations, bool annou
 future<> migration_manager::push_schema_mutation(const gms::inet_address& endpoint, const std::vector<mutation>& schema)
 {
     netw::messaging_service::msg_addr id{endpoint, 0};
-    auto fm = std::vector<frozen_mutation>(schema.begin(), schema.end());
-    auto cm = std::vector<canonical_mutation>(schema.begin(), schema.end());
+    auto adjusted_schema = db::schema_tables::adjust_schema_for_schema_features(schema, get_local_storage_service().cluster_schema_features());
```

Comment:
Why do we need this change?

The "cdc" column should not be set until you can push it to all nodes.

Reply:
We still need to adjust the schema of the mutations we send, so ok.